app/controllers/providers/dwp/overrides_controller.rb

```diff
@@ -0,0 +1,82 @@
+module Providers
```

Comment:
Is it still our pattern to create request specs for new controllers? (or maybe we are now creating system specs instead?)